docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_osm_extension/_index.md

```diff
@@ -0,0 +1,143 @@
+
+## Create Open Service Mesh and Azure Monitor extension instances with sample app
+
+To create a new extension Instance, we will use the _k8s-extension create_ command while passing in values for the mandatory parameters. This scenario provides you with the automation to deploy the Open Service Mesh extension on your Azure Arc enabled Kubernetes cluster. In the screenshot below, notice how currently there are no extensions installed yet in your Arc enabled Kubernetes cluster.
+
+> **Note: Before installing the Open Service Mesh extension, make sure that the kubectl context is pointing to your Azure Arc enabled Kubernetes cluster. To do that, you can refer to the [official Kubernetes documentation] (https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) to find the options to change the kubecontext to different Kubernetes clusters.**
+
+    ![Screenshot showing current kubectl context pointing to CAPI cluster](./01.png)
+
+    ![Screenshot showing Azure Portal with Azure Arc enabled Kubernetes resource extensions](./02.png)
+
+* Edit the environment variables in the [_capi_osm_extension.sh_](https://github.com/main/azure_arc/blob/main/azure_arc_k8s_jumpstart/cluster_api/capi_osm_extension/capi_osm_extension.sh) shell script to match your environment parameters followed by running the ```. ./capi_osm_extension.sh``` command.
+
+    ![Screenshot parameter examples](./03.png)
+
+ > **Note: The extra dot is due to the shell script has an *export* function and needs to have the vars exported in the same shell session as the rest of the commands.**
+
+   The script will:
+
+* Login to your Azure subscription using the SPN credentials
+* Add or Update your local _connectedk8s_ and _k8s-extension_ Azure CLI extensions
+* Create Open Service Mesh k8s extension instance
+* Create Azure Monitor k8s extension instance
+* Download and install OSM cli locally
+* Create four namespaces in kubernetes to deploy a test app
+* Onboard the Namespaces to the OSM Mesh and enable sidecar injection on the namespaces
+* Enable metrics for pods belonging to app namespaces
+* Update the namespaces to be monitored by modifying the configmap provided by the OSM
+* Deploy the apps to the namespaces
+
+You can now see that Open Service Mesh & Azure Monitor extensions are enabled once you visit the extension tab section of the Azure Arc enabled Kubernetes cluster resource in Azure.
+
+![Screenshot extension deployment security tab](./04.png)
+
+* You can also verify the deployment by running the kubectl command to ge look at the deployed artifacts in the arc-osm-system namespace below:
+
+![Screenshot extension deployment on cluster](./05.png)
+
+After 15 minutes or so you can verify the integration and moniotring insights coming from OSM to Azure Monitor by following the below steps in Azure portal.
+
+Verify the namespaces are showing up in the insights section of the Arc enabled Kuberentes resource in Azure portal.
```

Comment:
```suggestion
You can now see that Open Service Mesh & Azure Monitor for containers extensions are now enabled in the extension tab section of the Azure Arc enabled Kubernetes cluster resource in Azure.

![Screenshot extension deployment security tab](./04.png)

* You can also verify the deployment by running the below _kubectl_ commands and see the deployed artifacts in the _arc-osm-system_ namespace.

![Screenshot extension deployment on cluster](./05.png)

* It can take ~15 minutes before you will be able to verify the integration and monitoring insights coming from OSM to Azure Monitor by following the below steps in the Azure portal.

  *Verify the namespaces are showing up in the insights section of the Arc enabled Kubernetes resource in Azure portal.
```